devfile.yaml

```diff
@@ -1,19 +1,48 @@
 apiVersion: 1.0.0
 metadata:
-  name: che-in-che
+  name: che-server-in-che
 projects:
   - name: che
     source:
       type: git
       location: 'https://github.com/eclipse/che.git'
 components:
-  - alias: che-dev
-    type: kubernetes
-    reference: .che-dev.yaml
+  - type: dockerimage
+    image: 'eclipse/che-dev:nightly'
+    alias: che-server-dev
+    mountSources: true
+    memoryLimit: 5Gi
+    volumes:
+      - name: maven
+        containerPath: /home/user/.m2
+  - type: dockerimage
+    image: 'eclipse/che-server:nightly'
+    alias: che-server-runtime
+    mountSources: true
+    memoryLimit: 2Gi
```

Comment:
why we need 2Gi here?

Reply:
@skabashnyuk what amount of memory you think should be here?